tools/local-environment/tests/common.ts

```diff
@@ -25,7 +24,7 @@ export async function sendTokensAndConfirm(
   receiver: string,
   amounts: ethers.BigNumberish[],
   fastLiquidity = false
-): Promise<[boolean, ERC20]> {
+) {
```

Comment:
nit: return type `Promise<ERC20>`

Reply:
@anna-carroll  I removed it, as I have a dependency incomp. trying to import from `@nomad-xyz/contract-interfaces/bridge/ERC20`, while SDK's `.assetAtDestination()` return its own version and they don't match for some reason, I decided to natively pull type for the output and leave it for now